charts/service/ci/custom-values.yaml

```diff
@@ -97,3 +97,11 @@ hostAliases:
     hostnames:
       - anotherhost.com
       - anotherhost2.com
+
+persistentVolumeClaim:
+  - claimName: test-efs
```

Comment:
Do we want to keep this as test-efs?

Reply:
They're test values used by the ct command